### DISCUSSION TITLE
White browser window and PHP Fatal error in AbstractPluginExtension

### DISCUSSION THREAD

**Discussion by yweidmann** [kimai/kimai#3037] [Questions & Answers] [answered]
Dear list

I tried to run an export of certain data by using the export tool as PDF file (instead of Excel as usually, which works perfectly). From this moment on, I am getting a white browser-window when accessing kimai2 (e.g. login).

Based on the description, I tried to refresh the cache with
* `bin/console kimai:reload --env=prod`

As a result, I am getting the following return:
* `PHP Fatal error:  Class 'App\Plugin\AbstractPluginExtension' not found in /home/.../kimai2/var/plugins/EasyBackupBundle/DependencyInjection/EasyBackupExtension.php on line 18`

Seams, that something went wrong with the installation of the _EasyBackupBundle_ plugin.

Question:
* Is there a way to remove the _EasyBackupBundle_ plugin by using the console without harming the rest of the installation?
* Or any other suggestions?

Looking forward to any ideas and hints.

Best regards,
Yvo

**Comment by kevinpapst** [comment, marked as answer]
The class exists since two years. Weird... anyway, remove the bundle and recompile the cache:
```
rm -r var/plugins/EasyBackupBundle
bin/console kimai:reload --env=prod
```

**Reply by yweidmann** [reply]
Wow, that was a hyper-fast replay.

The `rm` worked perfectly:
* And the application works again.

 Thank you very much.

But I got a problem with `bin/console kimai:reload --env=prod` and getting the result:
`Command "kimai:reload" is not defined.
Did you mean one of these?
kimai:create-release
kimai:create-user
kimai:import-v1
kimai:install
kimai:plugins
kimai:version`

Is this problematic?

**Reply by kevinpapst** [reply]
How old is your Kimai setup? See `bin/console kimai:version`

**Reply by yweidmann** [reply]
`Kimai 2 - 1.3 stable (Ayumi) by Kevin Papst and contributors.`

**Reply by kevinpapst** [reply]
Hahaha, no wonder it doesn't work. That is a real case of "never touch a running system".
<img width="990" alt="Bildschirmfoto 2021-12-20 um 17 51 50" src="https://user-images.githubusercontent.com/533162/146803215-453623b8-46f0-44cd-8faf-cc728333421b.png">
Sorry, the docs assume you are up-to-date and not on a 2 year old system.

**Reply by yweidmann** [reply]
That is fully OK for. Thank you very much for your time.

But this leads to a further question regarding my old setup:
* I would like to get it updated. But I am not fit in Linux- and Web-stuff.
* Probably, the installation was not done the best way either.
* Some years ago, you offered on your website paid support. Is this still the case?
* Or would you propose to migrate to the [cloud](https://www.kimai.cloud/)?

**Reply by kevinpapst** [reply]
Yes https://www.kimai.org/store/installation-support.html and maybe. Lets talk via email https://www.kevinpapst.de/kontakt/
